### PR TITLE
Some small corrections and ideas for the German translation

### PR DESCRIPTION
**Pull request by monnypython**

### CODE REVIEW

**Comment by eikek** [comment]
Thanks so much! I'

File: modules/webapp/src/main/elm/Messages/Comp/ClassifierSettingsForm.elm [hunk 1]
```diff
@@ -63,8 +63,8 @@ sollen (blacklist) oder ausschließlich gelernt werden sollen
 Eine leere Whitelist stellt das Auto-Tagging ab.
 """
     , blacklistOrWhitelist = "Ist das folgende eine Blacklist oder eine Whitelist?"
-    , whitelistLabel = "Nur aus diesen Tag-Katogrien lernen"
-    , blacklistLabel = "Aus diesen Tag-Kategorien nicht lernen"
+    , whitelistLabel = "EINschließen der Tag-Kategorie"
+    , blacklistLabel = "AUSschließen der Tag-Kategorie"
```
[the remark this answers is not in the document]
Comment (monnypython) [reply modules/webapp/src/main/elm/Messages/Comp/ClassifierSettingsForm.elm:67]: Yes! "Kategorien" make sense here.

File: modules/webapp/src/main/elm/Messages/Comp/AttachmentMeta.elm [hunk 2]
```diff
@@ -54,11 +54,11 @@ de =
...
     , labels = "Label"
     , proposals = "Vorschläge"
-    , correspondentOrg = "Korrespondent Organisation"
-    , correspondentPerson = "Korrespondent Person"
-    , concerningPerson = "Betreffend Person"
-    , concerningEquipment = "Betreffend Zubehör"
-    , itemDate = "Dokument Datum"
-    , itemDueDate = "Fälligkeits-Datum"
+    , correspondentOrg = "Absender Unternehmen"
+    , correspondentPerson = "Absender Person"
```
[the remark this answers is not in the document]
Comment (monnypython) [reply modules/webapp/src/main/elm/Messages/Comp/AttachmentMeta.elm:58]: I didn't think about that. Correspondent is probably the best description here 👍🏼

File: modules/webapp/src/main/elm/Messages/Comp/AttachmentMeta.elm [hunk 3]
```diff
@@ -54,11 +54,11 @@ de =
...
+    , concerningPerson = "Empfänger Person"
```
[the remark this answers is not in the document]
Comment (monnypython) [reply modules/webapp/src/main/elm/Messages/Comp/AttachmentMeta.elm:59]: Yes, i think "Betreffende Person" is a good choice here.

File: modules/webapp/src/main/elm/Messages/Comp/ClassifierSettingsForm.elm (hunk 1 above)
[the remark this answers is not in the document]
Comment (eikek) [reply modules/webapp/src/main/elm/Messages/Comp/ClassifierSettingsForm.elm:67]: The capitalization is ok to me.

**Review by eikek — COMMENTED** [review]
This looks good to me, thank you! I think the "Absender/Empfänger" in AttachmentMeta needs to be changed. If you're done, you can tell me so I can merge this.

**Review by eikek — APPROVED** [review]

**Pull request merged**